### PR TITLE
Functional scrolling wine component, placed all rendering tests into Render.jsx, moved App to components, index.jsx only renders App component, added rendering test for WinePairings component and RecipeNutritionModal, removed Image component from RecipeNutritionModal

### PR DESCRIPTION
**Pull request by careyl96**
Someone please review my pull request

### CODE REVIEW

**Comment by careyl96** [comment]
[PR for Carey's almost functional scrolling wine component](https://trello.com/c/OCv6AIHc/49-pr-for-careys-almost-functional-scrolling-wine-component)

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+  render() {
+    return (
+      <Grid>
+        <Row>
+          <Col sm={6} sm-push={6} className="recipe-right-container">
+            <Image src="https://media.blueapron.com/recipes/2901/square_newsletter_images/1523553529-1-0007-7392/0423_2PM_White-Cheddar-Burgers_5941_WEB_SQ_hi_res.jpg" className="recipe-image" />
+          </Col>
+          <Col sm={6} sm-pull={6} className="recipe-left-container">
+            <RecipeTitle />
```
Comment (jaweesner) [inline client/src/components/App.jsx:26]: would it be worthwhile to put these into separate grid rows?

File: client/src/components/RecipeServings.jsx [hunk 2]
```diff
@@ -4,7 +4,9 @@ const RecipeServings = () => (
   <li className="list-item-info">
     <img src="https://d1nhio0ox7pgb.cloudfront.net/_img/i_collection_png/256x256/plain/knife_fork.png" alt="" className="list-item-icon" />
     <div className="list-item-name">servings</div>
-    <span>2</span>
+    <div className="list-item-value">
+      <span>2</span>
```
Comment (jaweesner) [inline client/src/components/RecipeServings.jsx:8]: why is this in a span and not just the text '2'

File: client/src/components/WinePairings.jsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
+/* eslint-disable */
+import React from 'react';
+import $ from 'jquery';
```
Comment (jaweesner) [inline client/src/components/WinePairings.jsx:3]: pretty sure jquery should not be used in react

File: public/css/winePairings.css [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+}
+
+.recipe-wine-text {
+  float: left;
+  width: 70%;
+  padding-left: 20px;
+  white-space: normal;
+}
+
+.mb-5 {
```
Comment (jaweesner) [inline public/css/winePairings.css:57]: are you using bootstrap? this is built in if so.

**Review by jaweesner — APPROVED** [review]
Couple comments, but approving, no need to re-review

**Pull request merged**